pkg/config/config.go

```diff
@@ -70,6 +70,9 @@ type ContainersConfig struct {
 	// default for the runtime.
 	ApparmorProfile string `toml:"apparmor_profile"`
 
+	// Annotation to add to all containers
+	AdditionalAnnotations []string `toml:"additional_annotations"`
```

Comment:
I am late to the game: can we rename it to `annotations`? The `additional_` seems redundant.